### ISSUE TITLE
Dev test: Ratio and Proportion 1.0.0-dev.77

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#582]
@emily-phet , @terracoda , @Matthew-Moore240, @BLFiedler, @jbphet, ratio-and-proportion/1.0.0-dev.77 is ready for dev testing. This sim is trying to be published in early 2021 Q1, with RC testing later this month. Document issues in https://github.com/phetsims/ratio-and-proportion/issues and link to this
issue. This test is not delayed, but if testing doesn't start for some time, please let me know and I will get you the latest and greatest dev version.

Please note that there are known issues and bugs with this sim, see https://github.com/phetsims/ratio-and-proportion/issues?q=is%3Aissue+is%3Aopen+label%3Atype%3Abug for the current bugs. We wanted to expedite the testing even though these aren't yet fixed to expose as many problems as early as possible. 

Assigning @ariel-phet for prioritization.

<details>
<summary><b>General Dev Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Try to include browser version numbers
- If there is a console available, check for errors and include them in the Problem Description.
- Run through the string tests on at least one platform, especially if it is about to go to rc.
- Check the Game Up harness on one platform.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested,
list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>General Dev Test Platforms</h3>

- [x] Latest macOS, Chrome and Safari
- [x] Latest iOS, Safari
- [x] Windows 10, all browsers (IE dropped)
- [x] Latest Chrome OS, Chrome

<h3>Issues to Verify</h3>

- [x] [Both hands interaction sonification inconsistent during interview](https://github.com/phetsims/ratio-and-proportion/issues/230)

These issues should have either use the labels "status:ready-for-qa" or "status:ready-for-review." If it is ready for
QA then close the issue if fixed. If ready for review then leave open and assign back to the developer.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_all_phet.html)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) Dev Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature testing doesn't negatively affect the sim in any way.
- Load the a11y view and make sure that interacting with all elements in the simulation updates the appropriate
descriptions in the PDOM.

<h3>Focus and Special Instructions</h3>

[Provide further instructions here. If you have any further tests you want done or specific platforms you want tested,
list them here. Any behaviors you want QA to pay special attention to should be listed here.]

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_all_phet.html)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen
readers. If you simply need a refresher on screen readers, please consult the
[QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md), which should have all of
the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view
before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as
intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in
[this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a
screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a
screen reader.

<h3>Magnification</h3>

This sim supports magnification with pinch and drag gestures on touch screens, keyboard shortcuts, and mouse/wheel controls. Please test magnfication and make sure
it is working as intended and well with the use cases of the simulation. Due to the way screen readers handle user input, magnification is NOT expected to
work while using a screen reader so there is no need to test this case.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>

  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary>

  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>

  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>

  We typically assign the developer who opened the issue in the QA repository.

  </details>

  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>

  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/documentation/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by zepumph** [comment]
There have already been a few minor bug fixes in the repo, please let me know when you are going to start testing this, and I will get a new dev version for you.

**Comment by zepumph** [comment]
I updated to dev.76

**Comment by zepumph** [comment]
After another few bug fixes and half of a code review in https://github.com/phetsims/ratio-and-proportion/issues/210, I updated the version to dev.77

**Comment by KatieWoe** [comment]
- [ ] Win 10 Chrome
- [ ] Win 10 Firefox
- [ ] Mac 11 Safari
- [ ] iPadOS 14 Safari
- [ ] ChromeOS

**Comment by zepumph** [comment]
@KatieWoe, I didn't say this explicitly (just via the sections I left in the template above). This sim supports the following a11y features that should be thoroughly tested:
* Full description with screen reader support
* Alternative input
* Sound/sonification
* Mobile description with iOS + VO support
* Zoom and Magnification

**Comment by KatieWoe** [comment]
Adding Screen Reader Checkmarks:
- [ ] Win 10 Chrome JAWS
- [ ] Win 10 Firefox NVDA
- [ ] MacOS 11 Safari VO
- [ ] iPadOS Safari VO

**Comment by zepumph** [comment]
@terracoda has been having a ton of trouble on mobile + VO on iOS 14. So it may be good to note potential trouble here. Though I don't know if there is an issue for this.

**Comment by KatieWoe** [comment]
@zepumph I'll keep an eye out. I noticed she heard something about it saying "sky" and I think I saw something like that during https://github.com/phetsims/QA/issues/581, but screen reader testing was cut short there so I didn't make an issue at the time.

**Comment by zepumph** [comment]
Ahh! That is great consistency to note (because of https://github.com/phetsims/ratio-and-proportion/issues/256).

https://github.com/phetsims/scenery/issues/1094 is the general blocking issue about a11y and iOS 14

**Comment by KatieWoe** [comment]
Memory test looks good:
<img width="409" alt="memtest" src="https://user-images.githubusercontent.com/41024075/102138793-27a30900-3e1a-11eb-8273-9471fe7ba4ae.png">

**Comment by terracoda** [comment]
In https://github.com/phetsims/QA/issues/582#issuecomment-743404519 @zepumph said:

> @terracoda has been having a ton of trouble on mobile + VO on iOS 14. So it may be good to note potential trouble here. Though I don't know if there is an issue for this.

My understanding was that there is a general issue for this that was opened with iOS 14 was launched. Issue https://github.com/phetsims/ratio-and-proportion/issues/256 seems to be specific to iOS 14, but https://github.com/phetsims/ratio-and-proportion/issues/283 is happening on iOS 13.

I think the general issue about iOS 14 is that context responses no longer fire when using iOS 14. I am not sure if any progress has been made on that general issue.

**Comment by zepumph** [comment]
>My understanding was that there is a general issue for this that was opened. .  .

See https://github.com/phetsims/scenery/issues/1094

**Comment by KatieWoe** [comment]
QA is done for the most part. I know @muedli is going to take one last look at Mac VO tonight, so I'll leave it to him to reassign and remove the tag.

**Comment by zepumph** [comment]
RC is is over in https://github.com/phetsims/QA/issues/601. So I'm going to close that. @muedli any testing you would like to do would be great over there :)

Thanks QA!
